stream_alert_cli/terraform_generate.py

```diff
@@ -601,16 +634,16 @@ def terraform_generate(**kwargs):
     with open('terraform/main.tf', 'w') as tf_file:
         tf_file.write(main_json)
 
-    # Break out early during the init process, clusters aren't needed yet
+    # Return early during the init process, clusters are not needed yet
     if init:
         return True
 
-    # Setup clusters
+    # Setup cluster files
     for cluster in config.clusters():
-        if cluster == 'main':
+        if cluster in RESTRICTED_CLUSTER_NAMES:
```

Comment:
You're now restricting using the cluster name of `athena` also - update the exceptions message to state the actual cluster name (formatted with `main` or `athena`)